src/lib/base/component-manager.js

```diff
@@ -64,6 +64,27 @@ const nonBubblingEventsMap = {
 // match & select "[eventType] [css selector]"
 const handlerMethodPattern = new RegExp(`^(${events.join('|')}|${Object.keys(nonBubblingEventsMap).join('|')}) (.*)`);
 
+/**
+ * Injects fallback mouse events for non-touch devices to events object.
+ * 
+ * @param {!Object} events
+ */
+function injectFallbackMouseEvents(events) {
+    const fallbackableEvents = ['touchend', 'tap'];
+
+    const _events = { ...events };
+
+    fallbackableEvents.forEach(eventType => {
+        if (events['click'] || !events[eventType]) {
```

Comment:
it looks like `if events['click']` exists we can already bypass this function? this might go on line 108?

Reply:
Yeah, I totally agree with this, it's not making any sense to have `events['click']` in the loop anyway.